scripts/utils/find_missing_source_tables.py

```
@@ -0,0 +1,138 @@
+            declared[(db, schema)].add(table["name"].upper())
+    return declared
+
+
+def fetch_db_tables(session: Session, database: str, schema: str) -> dict:
+    """Return {table_name: table_type} for all tables/views in a schema."""
+    query = f"""
+    SELECT table_name, table_type
+    FROM "{database}".INFORMATION_SCHEMA.TABLES
+    WHERE table_schema = '{schema}'
+    ORDER BY table_name
+    """
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)